test/integration/requirements.txt

```diff
@@ -0,0 +1,8 @@
+selenium==3.11.0
```

Comment:
Why the revert back to requirements.txt?

Reply:
I found runs to be about 15 seconds faster without pipenv. Unless there's a huge benefit to using it I'd prefer faster. I'm not very familiar with python's package management.